qemu/tests/cfg/rh_kselftests_vm.cfg

```diff
@@ -0,0 +1,11 @@
+- rh_kselftests_vm:
+    only RHEL
+    type = rh_kselftests_vm
+    virt_test_type = qemu
+    kernel_path = "/tmp/kernel"
+    kselftests_path = "/usr/libexec/kselftests"
+    variants:
+        - mm:
+            no s390x
```

Comment:
for s390x:
s390x:
                kvm_module_parameters = 'hpage=1'

Reply:
and cause 'hugetlb_fault_after_madv' in self test require 2M hugepage and s390x only support 1M hugepage, the skip is expected, please handle it and consider it as PASS with the final result

Reply:
the log is pasted in the jira task and for solving the issue I think either you could trigger the self test one by one and don't trigger 'hugetlb_fault_after_madv' for s390x or you could analysis the result and check if there's fail in the summary 'SUMMARY: PASS=9 SKIP=1 FAIL=0' @PaulYuuu @YongxueHong maybe you have better ideas?

Reply:
@zhencliu @yanan-fu it seems there could be a bug (which we need to discuss with the developer when he comes back), if the test is skipped the exit code is not 0 so the output of `run_vmtest.sh -t hugetlb` is interpreted as a failure, discussing with @fbq815 the possible workarounds could be:
- Executing the tests one by one in s390x excluding the `hugetlb_fault_after_madv`
- Try to handle the output looking for a skipped test in case of failure?

What do you think ? Thanks !

Reply:
Depends on how we handle the `SKIP`,   ignore it totally is risky IMO, I guess this is the reason that developer make it return false in this situation.  Agree to confirm with developer first.

If it is intentional,  we can use whitelist to list the tests which we know it can be skipped.

Reply:
Agreed with Yanan

Reply:
By the way,  i just checked the source code,   return code for `SKIP` is `4`.